packages/swingset/src/cli/commands/bootstrap.ts

```diff
@@ -0,0 +1,94 @@
+import fs from 'fs'
+import { Logs, codeText } from '../utils/logs'
+import { Bootstrap } from '../types'
+import childProcess from 'child_process'
+import { getPkgInstallCmd } from '../utils/get-pkg-install-cmd'
+import { FILES } from '../utils/constants'
+
+/*
+This command generates the following file structure, 
+Is supporting this old next versions aka page directory necessary? 
+app/
+(swingset)
+  ├ /layout.tsx
+  └ /swingset
+    ├ page.tsx 
+    └ /[...path]
+      └ page.tsx
+  next.config.js
+*/
+
+const bootstrap: Bootstrap = {
+  name: 'bootstrap',
+  description: 'Creates a swingset template in the `app` directory',
+  builder: {},
+  handler: async () => {
+    Logs.bootstrap.start()
+
+    const installSwingset = await getPkgInstallCmd()
+    console.log('Running', codeText(installSwingset))
+    const stdout = childProcess.execSync(installSwingset)
+    console.log(stdout.toString())
```

Comment:
Ideally we pipe the stdio from the child command, instead of buffering and logging (this should be the default).